server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/PortProvider.java

```diff
@@ -19,7 +19,7 @@
 
 package org.apache.james.webadmin;
 
-public interface Port {
+public interface PortProvider {
 
     int toInt();
```

Comment:
this method is not relevant anymore. generatePort ? or providePort ? But then, we should keep a Port object, right ?

Reply:
Generate a port in test
Provide a port in prod.

Seriously, I don't intend to change the way webadmin works (as I don't concider it bad,just it's naming could have been improved).